integrated_channels/sap_success_factors/client.py

```diff
@@ -144,6 +145,15 @@ def create_course_completion(self, user_id, payload):
         base_url = self.enterprise_configuration.sapsf_base_url
 
         if self.enterprise_configuration.prevent_self_submit_grades:
+            # TODO:
+            #   if/when we decide we should use the generic course completion path
+            #   for all customers, we should update the _payload_data method on the
+            #   SapSuccessFactorsLearnerDataTransmissionAudit class instead of doing this json load/dump
+            # Explanation:
+            #  json.loads loads "true" as True and when dumped again will be `true` (without quotes)
```

Comment:
OMG JSON loads why you do that.

Reply:
well in this case I want it to do that haha.
I originally had logic doing `courseCompleted = True if courseCompleted == "true" else False` and the test kept failing and that's when I took a closer look and realized just loading it was enough to get the boolean I wanted. 🙃 
I just felt the need to explain because I'm otherwise not doing anything here besides loading and re-dumping it lol.